guide_drafts/all-about-updates.md

```diff
@@ -0,0 +1,168 @@
+All About Updates
+=================
+
+Most applications fall into a category called "CRUD" apps. CRUD stands for Create Read Update Delete. Diesel provides support for all four pieces, but in this guide we're going to look at all the different ways to go about updating records with Diesel.
```

Comment:
You're right. I should mention that Diesel provides diesel as well. And also the things that Diesel provides.